src/Apps/MyCollection/Routes/PriceEstimate/PriceEstimateContactInformation.tsx

```diff
@@ -0,0 +1,189 @@
+  ContactInformationFormFragmentContainer,
+  ContactInformationFormModel,
+} from "Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm"
+import {
+  contactInformationValidationSchema,
+  validate,
+} from "Apps/Consign/Routes/SubmissionFlow/Utils/validation"
+import { useRequestPriceEstimate } from "Apps/MyCollection/Routes/PriceEstimate/Mutations/useRequestPriceEstimate"
+import { BackLink } from "Components/Links/BackLink"
+import { MetaTags } from "Components/MetaTags"
+import { Form, Formik } from "formik"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useTracking } from "react-tracking"
+import { RouterLink } from "System/Router/RouterLink"
+import { useRouter } from "System/Router/useRouter"
+import { PriceEstimateContactInformation_artwork$data } from "__generated__/PriceEstimateContactInformation_artwork.graphql"
+import { PriceEstimateContactInformation_me$data } from "__generated__/PriceEstimateContactInformation_me.graphql"
+
+const getContactInformationFormInitialValues = (
+  me: PriceEstimateContactInformation_me$data
+): ContactInformationFormModel => ({
+  name: me?.name || "",
+  email: me?.email || "",
+  phone: {
+    isValid: !!me?.phoneNumber?.isValid,
+    national: me?.phoneNumber?.national ?? undefined,
+    international: me?.phoneNumber?.international ?? undefined,
+    regionCode: me?.phoneNumber?.regionCode ?? undefined,
+  },
+})
+
+export interface PriceEstimateContactInformationProps {
+  artwork: PriceEstimateContactInformation_artwork$data
+  me: PriceEstimateContactInformation_me$data
+}
+
+export const PriceEstimateContactInformation: React.FC<PriceEstimateContactInformationProps> = ({
+  artwork,
+  me,
+}) => {
+  const { sendToast } = useToasts()
+  const { router } = useRouter()
+  const { trackEvent } = useTracking()
+  const { submitMutation } = useRequestPriceEstimate()
+
+  const initialValue = getContactInformationFormInitialValues(me)
+  const initialErrors = validate(
+    initialValue,
+    contactInformationValidationSchema
+  )
+
+  const handleSubmit = async ({
+    name,
+    email,
+    phone,
+  }: ContactInformationFormModel) => {
+    try {
+      trackEvent({
+        action: ActionType.sentRequestPriceEstimate,
+        artwork_id: artwork.internalID,
+        artwork_slug: artwork.slug,
+        user_id: me?.internalID,
+        user_email: me?.email,
+      })
+
+      await submitMutation({
+        variables: {
+          input: {
+            artworkId: artwork.internalID,
+            requesterEmail: email,
+            requesterName: name,
+            requesterPhoneNumber: phone.international,
+          },
+        },
+      })
+
+      router.push(`/my-collection/artwork/${artwork.internalID}`)
+    } catch (error) {
+      console.error(error)
+
+      sendToast({
+        variant: "error",
+        message: "An error occurred",
+        description: "Please contact sell@artsymail.com",
+      })
+
+      return
```

Comment:
**Non-Blocking**: `return` may be unnecessary here 🤔

Reply:
That's true, I'll remove the return 👍